src/main/python/generateStatisticForUpsertS3File.py

```diff
@@ -0,0 +1,29 @@
+import json
+import boto3
+import csv
+import io
+import datetime
+import botocore
+
+s3Client = boto3.client('s3')
+
+def lambda_handler(event, context):
+    print(event)
+    print(context)
+    bucket = event['Records'][0]['s3']['bucket']['name']
+    key = event['Records'][0]['s3']['object']['key'].rsplit('.', 1)[0]
+    print(bucket)
+    print(key)
+
+    key_status = key + '.status'
+    try:
+        response = s3Client.get_object(Bucket=bucket, Key=key_status)
+    except botocore.exceptions.ClientError as e:
```

Comment:
it can be bigger than just file missing. what about 403 when you don't have perms?